src/app/index.js

```diff
@@ -1,117 +1,107 @@
-const gui = require('nw.gui');
 const gkm = require('gkm');
-const pjson = require('../package.json');
 const AudioStreamMeter = require('audio-stream-meter');
-const character = document.getElementById("character");
-var exportable = require("./twitchAuth.js");
 
-//Tesing twitch auth file
-exportable.doSomething();
+const exportable = require('../app/twitchAuth.js');
+
+const character = document.getElementById('character');
+
+// Tesing twitch auth file
+exportable.doSomething(window);
 
 // Set window to always on top (This helps with some games)
-gui.Window.get().setAlwaysOnTop(true);
+nw.Window.get().setAlwaysOnTop(true);
 
 // Get app version
-document.getElementById("app-title").innerText += " " + pjson.version;
+document.getElementById('app-title').innerText += ' ' + nw.App.manifest.version;
 
 // === Decibel Meter Logic ===
 navigator.mediaDevices.getUserMedia({audio:true})
-.then(stream => {
-    var audioContext = new AudioContext();
-    var mediaStream = audioContext.createMediaStreamSource(stream);
-    var audioConfig = { volumeFall: 0.2 };
-    
-    var meter = AudioStreamMeter.audioStreamProcessor(audioContext, function() {
+  .then((stream) => {
+    let audioContext = new AudioContext();
+    let mediaStream = audioContext.createMediaStreamSource(stream);
+    let audioConfig = { volumeFall: 0.2 };
+
+    let meter = AudioStreamMeter.audioStreamProcessor(audioContext, function() {
         volume.style.width = meter.volume * 100 + '%';
         volume.innerText = meter.volume.toString();
 
         if (meter.volume >= 0.3) {
-          character.src = "../images/pigeon5.png";
-        }
-        else if (meter.volume >= 0.15) {
-          character.src = "../images/pigeon4.png";
-        }
-        else if (meter.volume >= 0.07) {
-          character.src = "../images/pigeon3.png";
-        }
-        else if (meter.volume >= 0.02) {
-          character.src = "../images/pigeon2.png";
-        }
-        else {
-          character.src = "../images/pigeon1.png";
+          character.src = '../images/pigeon5.png';
+        } else if (meter.volume >= 0.15) {
+          character.src = '../images/pigeon4.png';
+        } else if (meter.volume >= 0.07) {
+          character.src = '../images/pigeon3.png';
+        } else if (meter.volume >= 0.02) {
+          character.src = '../images/pigeon2.png';
+        } else {
+          character.src = '../images/pigeon1.png';
         }
     }, audioConfig);
-    
+
     mediaStream.connect(meter);
     stream.onended = meter.close.bind(meter);
 });
 
-// === WASD logic ===
-var top = 0;
-var left = 0;
-var keyList = [];
-
-gkm.events.on('key.*', function (data) {
+function wasdLogic () {
+  let top = 0;
```

Comment:
`top` was already globally defined, so I wrapped this code in a function so it can be created as a variable here using `let`